tweak Geant4ToolchainBackwardCompatibility.cmake to get the envvars we want. bug 409853
diff --git a/cmake/Modules/Geant4ToolchainBackwardCompatibility.cmake b/cmake/Modules/Geant4ToolchainBackwardCompatibility.cmake
index 41a88c4..e93b923 100644
--- a/cmake/Modules/Geant4ToolchainBackwardCompatibility.cmake
+++ b/cmake/Modules/Geant4ToolchainBackwardCompatibility.cmake
@@ -678,15 +678,4 @@ foreach(_shell bourne;cshell)
     @ONLY
     )
 
-  # Install it to the required location
-  install(FILES
-    ${PROJECT_BINARY_DIR}/InstallTreeFiles/${_scriptfullname}
-    DESTINATION ${CMAKE_INSTALL_BINDIR}
-    PERMISSIONS 
-    OWNER_READ OWNER_WRITE OWNER_EXECUTE
-    GROUP_READ GROUP_EXECUTE
-    WORLD_READ WORLD_EXECUTE
-    COMPONENT Runtime
-    )
 endforeach()
-
diff --git a/cmake/Templates/geant4-env-skeleton.in b/cmake/Templates/geant4-env-skeleton.in
index ea8f104..db68b9c 100644
--- a/cmake/Templates/geant4-env-skeleton.in
+++ b/cmake/Templates/geant4-env-skeleton.in
@@ -1,30 +1,3 @@
-#!@GEANT4_TC_SHELL_PROGRAM
-
-#----------------------------------------------------------------------------
-# @GEANT4_TC_SHELL_FAMILY@ environment setup script for Geant4 @Geant4_VERSION@
-#
-# This script will configure your environment so that any Geant4 supplied
-# tools, libraries and data are available in your PATH, library paths etc.
-# Data libraries will only be setup if they were installed as part of the
-# Geant4 installation.
-#
-# Source this script to perform the setup.
-#
-# This script is autogenerated by CMake DO NOT EDIT
-#
-
-#----------------------------------------------------------------------------
-# Locate directory of self
-#
-@GEANT4_ENV_SELFLOCATE_COMMAND@
-
-
-#----------------------------------------------------------------------------
-# Setup binary and library paths...
-#
-@GEANT4_ENV_BINPATH_SETUP@
-@GEANT4_ENV_LIBPATH_SETUP@
-
 #----------------------------------------------------------------------------
 # Resource file paths
 # Data Library Paths are only set if they were installed by CMake
